### PR TITLE
Readme and examples

### PR DESCRIPTION
**Pull request by whyoleg**
* Update readme
* Add 3 examples: multiplatform chat, nodejs transport, interactions

(Now base branch it `task/interactions-api`, can be merged after it will be merged)

### CODE REVIEW

**Comment by yschimke** [comment]
Only skimmed as sample code, but LGTM

File: README.md [hunk 1]
```diff
@@ -1,2 +1,221 @@
...
+
+- request/response (stream of 1)
+- request/stream (finite stream of many)
+- fire-and-forget (no response)
+- event subscription (infinite stream of many)
+
+Learn more at http://rsocket.io
+
+## Supported platforms and transports :
+Transports based on [ktor](https://github.com/ktorio/ktor), so it depends on `ktor` engines what supported and what not:
```
Comment (yschimke) [inline README.md:16]:
```suggestion
Transports are implemented based on [ktor](https://github.com/ktorio/ktor) to ensure Kotlin multiplatform. So it depends on `ktor` engines for available transports and platforms (JVM, JS, Native):
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,221 @@
...
+  `fun requestStream(payload: Payload): Flow<Payload>`
+* Request-Channel: 
+
+  `fun requestChannel(payloads: Flow<Payload>): Flow<Payload>`
+* Metadata-Push:
+  
+  `suspend fun metadataPush(metadata: ByteReadPacket)`
+
+## Using in your projects
+The `master` branch is now dedicated to development of multiplatform reborn of rsocket-kotlin.
```
Comment (yschimke) [inline README.md:41]:
```suggestion
The `master` branch is now dedicated to development of multiplatform rsocket-kotlin.
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,221 @@
...
+
+## Usage
+
+### Client WebSocket with CIO ktor engine
+
+```kotlin
+//create ktor client
+val client = HttpClient(CIO) {
+    install(WebSockets)
+    install(RSocketClientSupport) {
```
Comment (yschimke) [inline README.md:115]: I think this is a ciritical point to show the configuration

```
        payloadMimeType = PayloadMimeType("application/json",
          "application/json")

        this.setupPayload = setupPayload

        acceptor = {
          RSocketRequestHandler {
            requestResponse = {
              return it // echo payload
            }
...
          }
        }
        keepAlive = KeepAlive(5.seconds)
        if (debug) {
          plugin = plugin.copy(connection = listOf(::LoggingConnection))
        }
```

File: README.md [hunk 4]
```diff
@@ -1,2 +1,221 @@
...
+## Reactive Streams Semantics
+
+From [RSocket protocol](https://github.com/rsocket/rsocket/blob/master/Protocol.md#reactive-streams-semantics):
+
+    Reactive Streams semantics are used for flow control of Streams, Subscriptions, and Channels. 
+    This is a credit-based model where the Requester grants the Responder credit for the number of PAYLOADs it can send. 
+    It is sometimes referred to as "request-n" or "request(n)".
+
+`kotlinx.coroutines` doesn't truly support `request(n)` semantic, but it has `Flow.buffer(n)` operator
+which can be used to archive similar possibilities:
```
Comment (yschimke) [inline README.md:181]:
```suggestion
which can be used to achieve something similar:
```

File: README.md [hunk 5]
```diff
@@ -1,2 +1,221 @@
...
+//just after collection first request for 10 elements will be sent
+//after 10 elements collected, 10 more elements will be requested, and so on
+bufferedStream.collect { payload: Payload ->
+    println(payload.data.readText())
+}
+``` 
+
+## Bugs and Feedback
+
+For bugs, questions and discussions please use the [Github Issues](https://github.com/rsocket/rsocket-kotlin/issues).
```
Comment (yschimke) [inline README.md:205]: Slack?

File: README.md (hunk 3 above)
Comment (whyoleg) [reply README.md:115]: Yeah, Will add it

File: README.md (hunk 5 above)
Comment (whyoleg) [reply README.md:205]: I thought that rsocket slack isn't public, if so, does it make sense?

File: README.md (hunk 1 above)
Comment (whyoleg) [reply README.md:16]: Will apply suggested changes while updating readme with configuration setup

**Pull request merged**